pkg/k8s/ingress/crud_test.go

```diff
@@ -0,0 +1,112 @@
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	extensions "k8s.io/api/extensions/v1beta1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestCreate(t *testing.T) {
+	ctx := context.Background()
+	ingress := &extensions.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "fake",
+			Namespace: "test",
+		},
+	}
+
+	clientset := fake.NewSimpleClientset()
+	err := Create(ctx, ingress, clientset)
+	if err != nil {
+		t.Fatal(err)
+	}
+	retrieved, err := clientset.ExtensionsV1beta1().Ingresses(ingress.Namespace).Get(ctx, ingress.Name, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !reflect.DeepEqual(retrieved, ingress) {
+		t.Fatalf("Didn't created correctly")
+	}
+}
+
+func TestList(t *testing.T) {
+	ctx := context.Background()
+	ingress := &extensions.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "fake",
+			Namespace: "test",
+		},
+	}
+
+	clientset := fake.NewSimpleClientset(ingress)
+	ingressList, err := List(ctx, ingress.Namespace, "", clientset)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(ingressList) != 1 {
+		t.Fatal(fmt.Errorf("Expected 1 ingress, found %d", len(ingressList)))
+	}
+
+}
+
+func TestDestroy(t *testing.T) {
```

Comment:
It would be nice to cover the not-found error, and the generic error.

You have similar tests here:
https://github.com/okteto/app/blob/master/backend/pkg/k8s/services/crud_test.go#L151
https://github.com/okteto/app/blob/master/backend/pkg/k8s/services/crud_test.go#L196